TestFiles/bad.json

```diff
@@ -1,99 +1,72 @@
+  {"Path":"/api/actors?pageNumber=10.1","PerfTarget":{"Category":"SearchActors"},"Validation":{"StatusCode":400, "ContentType":"application/problem+json","jsonObject":[{"field":"type","value":"https://github.com/retaildevcrews/helium/blob/main/docs/ParameterValidation.md#actors"},{"field":"title","value":"Parameter validation error"},{"field":"detail","value":"One or more invalid parameters were specified."},{"field":"status","value":400},{"field":"instance","value":"/api/actors?pageNumber=10.1"},{"field":"validationErrors","Validation":{"jsonArray":{"count":1,"forEach":[{"jsonObject":[{"field":"code","value":"InvalidValue"},{"field":"target","value":"PageNumber"},{"field":"message","value":"The value '10.1' is not valid for PageNumber."}]}]}}}]}},
+  {"Path":"/api/actors?pageNumber=foo","PerfTarget":{"Category":"SearchActors"},"Validation":{"StatusCode":400, "ContentType":"application/problem+json","jsonObject":[{"field":"type","value":"https://github.com/retaildevcrews/helium/blob/main/docs/ParameterValidation.md#actors"},{"field":"title","value":"Parameter validation error"},{"field":"detail","value":"One or more invalid parameters were specified."},{"field":"status","value":400},{"field":"instance","value":"/api/actors?pageNumber=foo"},{"field":"validationErrors","Validation":{"jsonArray":{"count":1,"forEach":[{"jsonObject":[{"field":"code","value":"InvalidValue"},{"field":"target","value":"PageNumber"},{"field":"message","value":"The value 'foo' is not valid for PageNumber."}]}]}}}]}},
+
+  {"Path":"/api/actors?pagenumber=0&pagesize=-1&q=a","PerfTarget":{"Category":"SearchActors"},"Validation":{"StatusCode":400, "ContentType":"application/problem+json","jsonObject":[{"field":"type","value":"https://github.com/retaildevcrews/helium/blob/main/docs/ParameterValidation.md#actors"},{"field":"title","value":"Parameter validation error"},{"field":"detail","value":"One or more invalid parameters were specified."},{"field":"status","value":400},{"field":"instance","value":"/api/actors?pagenumber=0&pagesize=-1&q=a"},{"field":"validationErrors","Validation":{"jsonArray":{"count":3,"forEach":[{"jsonObject":[{"field":"code","value":"InvalidValue"},{"field":"target"},{"field":"message"}]}]}}}]}},
```

Comment:
pagesize & pagenumber should be updated to pageSize and pageNumber. I ran this on helium-typescript, and it's case-sensitive